Justfile

```diff
@@ -10,33 +10,27 @@ deps:
     pnpm i --frozen-lockfile
 
 # Lint the workspace for all available targets
-lint: lint-specs-md-check lint-specs-toc-check lint-filenames lint-links lint-specs-spelling
+lint: lint-specs-md lint-specs-toc lint-specs-spelling lint-links lint-filenames
 
 # Updates all files to fix linting issues
-lint-fix: lint-specs-md-fix lint-specs-toc
+lint-fix: lint-specs-md-fix lint-specs-toc-fix lint-specs-spelling-fix
 
 # Validates markdown file formatting
-lint-specs-md-check:
+lint-specs-md:
     npx markdownlint-cli2 "./specs/**/*.md"
 
 # Updates markdown files formatting to satisfy lints
 lint-specs-md-fix:
     npx markdownlint-cli2 --fix "./specs/**/*.md"
 
 # Validates Table of Content Sections with doctoc
-lint-specs-toc-check:
+lint-specs-toc:
     npx doctoc '--title=**Table of Contents**' ./specs && git diff --exit-code ./specs
 
 # Updates Table of Content Sections with doctoc
-lint-specs-toc:
+lint-specs-toc-fix:
     npx doctoc '--title=**Table of Contents**' ./specs
```

Comment:
I'm not sure why we're changing the meaning of existing targets here - that will just break things for people running them locally.  And the shorter name for the fix is probably better given that's what people running manually probably want to do and only CI really needs the -check variant.

Reply:
I'm fine with using the shorter name for fix, and a longer name for checks. Next commit does that.

What I'm trying to fix here is that we use all possible combinations for naming according to fix/check, which I found quite confusing when trying to work out how this worked. Currently:
- `lint`/`lint-fix`
- `lint-specs-md-check`/`lint-specs-md-fix`
- `lint-specs-toc-check`/`lint-specs-toc`
- `lint-specs-spelling`/`lint-specs-spelling-fix`